config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   resources :contacts, only: [:index]
   resources :users, only: [:index, :new, :create, :destroy]
+  passwordless_for :users, at: '/', as: :auth
```

Comment:
We are going to need to secure all routes I guess, but we will also require a way to login for the first users.  A database seed script with key email addresses for users in production.

Reply:
Captured in trello task